app/views/deploys/recent.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1><%= title %></h1>
 
 <div class="timeline" ng-controller="<%= controller %>">
+  <input type=hidden  id=default_time_format value="<%= current_user.time_format %>">
```

Comment:
maybe `checked="cachekd"` ... but totally possible that this won't work out ... angular might want to set/unset these boxes :(